index.js

```diff
@@ -6,24 +6,24 @@ gameStartElement.addEventListener('click', (event) => {
 
 document.addEventListener("keydown", event => {
```

Comment:
There' a lot of repetition `document.addEventListener("keydown" ...` 
Try to remove it.